manual/1password4/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,43 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName    = $env:ChocolateyPackageName
+  fileType       = 'exe'
+  url            = ''
+  softwareName   = '1Password*'
+  checksum       = ''
+  checksumType   = 'sha256'
+  silentArgs     = "/VERYSILENT /NORESTART /SUPPRESSMSGBOXES /SP- /LOG=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).InnoInstall.log`""
+  validExitCodes = @(0)
```

Comment:
Reminder to myself, the exit codes need to be updated to confirm to our `Pending` guidelines.